core/platform/email/gae_email_services.py

```diff
@@ -20,9 +20,13 @@
 
 from google.appengine.api import mail
 
+def get_reply_to_email_with_id(reply_to_id):
```

Comment:
Add one more newline above this one.

How about _get_incoming_email_address(reply_to_id)?